scripts/setup.js

```diff
@@ -1,8 +1,12 @@
-/* eslint-disable no-console */
-import redb from 'rethinkdbdash';
-import { database } from '../config';
+require('dotenv').config();
+const redb = require('rethinkdbdash');
+
+const credentials = {
```

Comment:
can we just remove this variable and put credentials right into the constructor on L10?